### ISSUE TITLE
Model vs Obs Lat Lon plots of U200

### ISSUE THREAD

**Issue by JulioTBacmeister** [NCAR/ADF#246]
### New feature type

New plot and/or plot enhancement

### What is this new feature?

I'd like maps of U200 (Zonal wind at 200 hPa) vs reanalyses - ERA5 or MERRA would be fine.  

### Assistance required?

Yes, I am requesting a new feature but won't have time to develop it

**Comment by brianpm** [comment]
I recently added monthly mean U from ERA5, and it has 200hPa. So this should be working already, I think. Maybe @justin-richling can check if it actually does work.

**Comment by justin-richling** [comment]
Indeed, it does appear that [U200](https://project.cgd.ucar.edu/projects/ADF/tests/U200/html_img/plot_page_U_200hpa_ANN_LatLon_Mean.html) works with the new obs @brianpm put in recently.

**Comment by nusbaume** [comment]
@JulioTBacmeister Are you just wanting to compare the zonal winds between CAM and ERA5 at 200 hPa using the `U` variable from CAM?  If so then @brianpm is correct that we already have that functionality.

However, if you are looking specifically for a comparison between the `U200` CAM output variable and ERA5 or MERRA then yes, it looks like we'll need to create a new ADF variable defaults entry and ERA5 obs variable in order to do so.

**Comment by brianpm** [comment]
@nusbaume -- That's right, I didn't think about that distinction (U200 in the CAM output vs specifying the 200hPa level with `plot_press_levels`.

**Comment by JulioTBacmeister** [comment]
Thanks. This looks good.  I just wanted a comparison of U200 from CAM against any reanalysis. I'm not picky.  It would also be nice to add U200 to the exp vs CTL plots.
